src/features/contact/shared.ts

```diff
@@ -23,7 +23,7 @@ export const mapKlantToKlantIdentifier = (
         bsn: klant.bsn,
         vestigingsnummer: klant.vestigingsnummer,
         kvkNummer: klant.kvkNummer,
-        rsin: klant.rsin,
+        rsin: klant.rsin ?? klant.nietNatuurlijkPersoonIdentifier,
```

Comment:
kunnen we als onderdeel van de refactoring overzichtelijker gebruik maken van deze velden? zie andere comment over een kvknummer in een rsin

Reply:
het op zich out of scope en ik ben bang dat het analyseren van eventuele testbevindingen extra lastig wordt als we dit ook nog gaan aanpassen. Dus ik zou dit liever voor nu even laten

Reply:
zie andere comment. aanpassingen staan klaar.